pkg/controller/flash/flash.go

```diff
@@ -85,6 +85,13 @@ func (f *Flash) Clear() {
 	delete(f.values, flashKeyWarning)
 }
 
+// Clone makes a copy of this flash data into the new target.
+func (f *Flash) Clone(values map[interface{}]interface{}) {
```

Comment:
it feels weird that the target map is passed in

Reply:
That's how all flashes are created: https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/controller/flash.go#L29

It's to avoid a circular dependency on session depending on flash depending on session.